mod.d/openssh.yaml

```diff
@@ -1527,6 +1527,7 @@ content: !!str |
 
       # 3.  Existence of the pieces of the authorized_key path for each user.
       # 4.  Mode and UID of the relative authorized_key paths for each user.
+      excluded_usernames = {"ssm-user"} # Exclude ssm-user from check as it is not accessed via SSH
```

Comment:
Can we move this to the config dictionary?

Reply:
Sure.

I've got no idea how to properly unit test this though, tried for a few hours and never got it working.

Reply:
You could probably copy test_ssh_get_dag and add the ssm-user to the glob-mock. Since the user is supposed to be skipped, the DAG should be identical to the copy defined in test_ssh_get_dag.

Reply:
Done and done, I believe!